PKGBUILD

```diff
@@ -124,15 +133,15 @@ build() {
 
   export CC=clang
   export CXX=clang++
-  export AR=llvm-ar
-  export NM=llvm-nm
+  export AR=ar
+  export NM=nm
```

Comment:
Would this be confused with GCC's `ar` and `nm` instead of LLVM's?

Reply:
Arch uses GCC's ```ar``` and ```nm```, not LLVM's

Reply:
Are they also using Clang and LLD? Why would they mix GCC and LLVM compilation tooling like this?

Reply:
Yes, they are combining clang and gcc stuff, dunno why, tried searching their commits for a reason but nothing appeared, maybe I just didnt look hard enough

Reply:
Hmm, it's probably fine then. We could try asking the package maintainers too.